test/e2e/create_cluster_test.go

```diff
@@ -1169,6 +1168,12 @@ func TestCreateClusterV2(t *testing.T) {
 			obj.Annotations[hyperv1.ControlPlaneOperatorV2Annotation] = "true"
 		}
 	}
+	if !e2eutil.IsLessThan(e2eutil.Version418) {
+		clusterOpts.FeatureSet = string(configv1.TechPreviewNoUpgrade)
```

Comment:
This is causing `TestCreateClusterV2` to fail on https://github.com/openshift/hypershift/pull/5514

I didn't realize we had copy/pastad the whole KAS config logic for v2